packages/Debian/armhf/DEBIAN/control

```diff
@@ -0,0 +1,10 @@
+Package: foglamp-service-notification
+Version: 1.0.0
+Section: devel
+Priority: optional
+Architecture: armhf
+Depends: foglamp
```

Comment:
`foglamp (>=1.5)`

Reply:
make_deb throws error on (>=1.5)

Reply:
At the moment you will need to create a foglamp debian with develop branch with the corresponding change in https://github.com/foglamp/FogLAMP/blob/develop/VERSION#L1 set to 1.5. As notification supports with v1.5 ONLY